packages/ddp-client/livedata_connection_tests.js

```diff
@@ -95,6 +95,92 @@ Tinytest.add("livedata stub - receive data", function (test) {
   test.isUndefined(conn._updatesForUnknownStores[coll_name]);
 });
 
+Tinytest.add("livedata stub - buffering data", function (test) {
```

Comment:
Done.  The client test was failing when it was run in the browser because of the requirement on `lolex` and the (previous) inability to `Npm.require` in the client.  Now using `ecmascript` and `import` it works perfect.